server/utils/applications/submittedApplicationSummaryCards.ts

```diff
@@ -0,0 +1,77 @@
+import { ApprovedPremisesApplication as Application } from '../../@types/shared'
+import { FormSections, SummaryListItem, UiTask } from '../../@types/ui'
+import Apply from '../../form-pages/apply'
+import { documentsFromApplication } from '../assessments/documentUtils'
+import { getActionsForTaskId } from '../assessments/getActionsForTaskId'
+import { linebreaksToParagraphs } from '../utils'
+import { embeddedSummaryListItem } from './summaryListUtils'
+
+type QuestionResponse = string | Array<Record<string, unknown>>
+
+type PageResponse = Record<string, QuestionResponse>
+
+type TaskResponse = Array<PageResponse>
+
+export class SumbmittedApplicationSummaryCards {
+  constructor(
+    private readonly application: Application,
+    private readonly assessmentId?: string,
+    private readonly sections: FormSections = Apply.sections.slice(0, -1), // Defaults to all Apply sections except the final "Check your answers" section
```

Comment:
Not for now but I think we're doing `Apply.sections.slice(0, -1)` in a couple places, we could save it as a `const` and use that instead